src/hidimstat/knockoffs.py

```diff
@@ -361,6 +394,71 @@ def fdr_selection(
         return selected
 
 
+def job_lib_lasso_statistic(X, X_tilde, y, test_linear_model, test_preconfgure_model):
```

Comment:
For simplification, can we make this a class a function and use self.test_linear_model, self.test_preconfgure_model instead of having them as arguments of the function.

Reply:
For example, if we use the same model/object for all the jobs to fit, it will be problematic because there are multiple fit at the same time.

Reply:
This could be solved with cloning objects and joblib's documentation offers solution for this type of situation. 

But I see your point. Could we make it a static_method for the moment, to still reflect that it is only used in the class.

Reply:
see #513